src/modules/ProductLicense.rb

```diff
@@ -380,20 +380,14 @@ def GetLicenseDialog(languages, license_language, licenses, id, spare_space)
                 Id(:printing_hint),
                 Label(
                   @license_on_installed_system ?
-                    # TRANSLATORS: addition license information
-                    # %s is replaced with the directory name
-                    format(
-                      _("This EULA can be found in the directory %s"),
-                      @license_file_print
-                    ) :
-                    # TRANSLATORS: addition license information
-                    # %s is replaced with the filename
-                    format(
-                      _(
-                        "If you want to print this EULA, you can find it\non the first media in the file %s"
-                      ),
-                      @license_file_print
-                    )
+                      # TRANSLATORS: addition license information
+                      # %s is replaced with the directory name
+                      _("This EULA can be found in the directory %s") % @license_file_print
```

Comment:
NP: As currently discussed on yast-devel@, I'd put the directory name on a separate line to make sure it fits the dialog in text mode.